app/views/shared/_announcement.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="announcement container-fluid bg-light">
   <div class="row align-items-center">
     <div class="col align-content-center">
-      <p class="text-center mt-2 mb-2 pb-1"><i class="fas fa-bullhorn"></i><%= blackcat_config.blackcat_message(:announcement) %></p>
+      <p class="text-center mt-2 mb-2 pb-1"><i class="fas fa-bullhorn"></i> <%= ENV['announcement'] || t('blacklight.announcement.html') %></p>
```

Comment:
```suggestion
      <p class="text-center mt-2 mb-2 pb-1"><i class="fas fa-bullhorn"></i> <%= sanitize ENV['announcement'] || t('blacklight.announcement.html') %></p>
```

Reply:
above change because 

[](url)
![Screen Shot 2020-04-08 at 1 10 56 PM](https://user-images.githubusercontent.com/6477867/78813785-89fa5e80-799b-11ea-81c7-b0ba89aa4ef1.png)

Reply:
Thanks, this is resolved in latest